src/views/apply/index.html

```diff
@@ -3,6 +3,19 @@
 {% set homeLink = true %}
```

Comment:
Would it be an idea to remove the home link since this goes back to the homepage?

Reply:
The links are for different audiences and have different purposes. One is for people familiar with the website who want to go home, the other is for people unfamiliar with the website who want to find out more info. I think we need both.